<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_export_moreFile_data</class>
 <widget class="QDialog" name="dlg_export_moreFile_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_read">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>558</width>
     <height>235</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>导出文件</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="m_lbName">
        <property name="text">
         <string>存储版文件路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_leName_3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="m_lbLocalPath">
        <property name="text">
         <string>本地文件路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_leLocalPath_3"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnExport">
        <property name="text">
         <string>选择本地文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QComboBox" name="m_cboxExportType"/>
      </item>
      <item>
       <widget class="QPushButton" name="pbtn_read">
        <property name="text">
         <string>开始导出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
